### DISCUSSION TITLE
What if Generic 900 True Diversity PWM 16.json soft-bricked?

### DISCUSSION THREAD

**Discussion by lida2003** [ExpressLRS/ExpressLRS#2371] [Q&A] [answered]
Hi guys,

I found [src/hardware/RX/Generic 900 True Diversity PWM 16.json](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/hardware/RX/Generic%20900%20True%20Diversity%20PA.json) , which didn't has "button" definition. It used for PWM output.

So what if Generic 900 True Diversity PWM 16.json soft-bricked?

**Comment by pkendall64** [comment, marked as answer]
A button definition just means that the button is usable in ELRS. Even without a button definition you can hold GPIO0 low and put the chip into downloader mode.

**Reply by lida2003** [reply]
OK. It helps.
